tasks/sep/021-stage-1-4/README.md

```diff
@@ -0,0 +1,37 @@
+# Stage 1.4 Upgrade
+
+Status: DRAFT, NOT READY TO SIGN
+
+## Objective
+
+Registers a new `FaultDisputeGame` with the `DisputeGameFactory` that is backed by the [`asterisc`][asterisc] FPVM and
+the [`kona`][kona] fault proof program.
+
+This upgrades the Fault Proof contracts in the
+[op-contracts/v1.9.0-rc.3](https://github.com/ethereum-optimism/optimism/tree/op-contracts/v1.9.0-rc.3) release.
+
+## Pre-deployments
+
+- `RISCV` - `0x7f9a34da8d240bbe039e9f7b497c9bc500f6acf5`
+- `DelayedWETHProxy` - `0x3c024f698c27a914350531bc48033e62d42703e0`
```

Comment:
The version here doesn't seems to be the [v.1.9.0-rc.3](https://github.com/ethereum-optimism/optimism/blob/op-contracts/v1.9.0-rc.3/packages/contracts-bedrock/src/dispute/DelayedWETH.sol)
As the version is different [here](https://sepolia.etherscan.io/address/0x07f69b19532476c6cd03056d6bc3f1b110ab7538#code)
```solidity
 string public constant version = "1.1.0";
```
instead of [v.1.9.0-rc.3](https://github.com/ethereum-optimism/optimism/blob/op-contracts/v1.9.0-rc.3/packages/contracts-bedrock/src/dispute/DelayedWETH.sol)

```solidity
string public constant version = "1.2.0-beta.4";
```
Could we make sure we are deploying the exact same version for clarity? or instead adding the version used for this contract?